lldb/test/API/lua_api/TestLuaAPI.py

```diff
@@ -158,7 +158,11 @@ def get_tests(self):
         return tests
 
     def test_lua_api(self):
-        if "LUA_EXECUTABLE" not in os.environ or len(os.environ["LUA_EXECUTABLE"]) == 0:
+        if (
+            "LUA_EXECUTABLE" not in os.environ
+            or len(os.environ["LUA_EXECUTABLE"]) == 0
+            or "NOTFOUND" in os.environ["LUA_EXECUTABLE"]
+        ):
```

Comment:
How about we check if `os.path.exists(os.environ["LUA_EXECUTABLE"])` to cover both cases?

Reply:
yes, it's a nicest and more generic way, I will update the patch.

By the way,  while verifying the patch for the maybe-exotic corner case, I noticed that `LUAANDSWIG_FOUND` is incorrectly defined (to `1`) too as the `find_package_handle_standard_args(LuaAndSwig` call in `lldb/cmake/modules/FindLuaAndSwig.cmake` is not checking the `LUA_EXECUTABLE` variable (which is `LUA_EXECUTABLE-NOTFOUND` in this case. I think it is worth adding too.